optimum/onnxruntime/trainer.py

```diff
@@ -773,6 +780,13 @@ def evaluation_loop_ort(
             )
 
             logger.info("[INFO] Exporting the model to ONNX...")
+            if args.deepspeed and args.fp16:
+                warnings.warn(
```

Comment:
I wonder if we should check the `transformers` version and then raise a warning if the detected version doesn't match the required one for ONNX export on CUDA?

For now, this warning is OK but we'll probably want to revisit this once we bump `transformers` with your PR :)

Reply:
@lewtun Yes, exactly. I put it this way since I am not sure in which version of transformers will include it. Will check the transformers version once I have the information.

Reply:
Sounds good!